src/families/tron/bridge/js.js

```diff
@@ -71,6 +71,11 @@ const signOperation = ({ account, transaction, deviceId }) =>
           ? account.subAccounts.find(sa => sa.id === transaction.subAccountId)
           : null;
 
+      const balance =
+        subAccount ? subAccount.balance : account.spendableBalance;
```

Comment:
in case of parent account, isn't spendableBalance minus fees? that's what i did when implementing the "max spendable"

Reply:
you're right